packages/packagers/html/src/HTMLPackager.js

```diff
@@ -163,21 +163,20 @@ function insertBundleReferences(siblingBundles, tree) {
         tag: 'link',
         attrs: {
           rel: 'stylesheet',
-          href: urlJoin(
-            nullthrows(bundle.target).publicUrl,
-            nullthrows(bundle.name),
-          ),
+          href: urlJoin(bundle.target.publicUrl, bundle.name),
         },
       });
     } else if (bundle.type === 'js') {
       bundles.push({
         tag: 'script',
         attrs: {
           type: bundle.env.outputFormat === 'esmodule' ? 'module' : undefined,
-          src: urlJoin(
-            nullthrows(bundle.target).publicUrl,
-            nullthrows(bundle.name),
-          ),
+          nomodule:
```

Comment:
Shouldn't this only be true if there is a corresponding module? (Not sure how to determine that)

Reply:
Maybe determine the main bundle in the bundle group, check if that script has nomodule, and also add nomodule to siblings in that case?

Reply:
I think the module and nomodule scripts all get placed in the same bundle group with the HTML...

Reply:
We may need to add an `asset.meta.nomodule` flag, and check the entry asset for that. Though not sure what about shared bundles... I guess we wouldn't want to make a shared bundle containing both `nomodule` and non-nomodule (i.e. still global output format but not from a nomodule source), but idk if this is even possible.